### ISSUE TITLE
no attribute 'CACHE_DIR' error when 'caching_enabled' False in python script

### ISSUE THREAD

**Issue by mayanklabs** [drien/tts-joinery#3]
Python Version: `3.10.11`

Python Script:
```python
from joinery.op import JoinOp
from joinery.api.openai import OpenAIApi

text_for_speech = """Testing the TTS functionality of Joinery with OpenAI API"""

tts = JoinOp(
    text=text_for_speech,
    api=OpenAIApi(
        model='tts-1',
        voice='alloy',
        caching_enabled=False
    )
)

tts.process_to_file('output.mp3')
```

Console Errors:
```bash
Traceback (most recent call last):
  File "c:\test\tts joiney\script.py", line 27, in <module>
    tts.process_to_file('output.mp3')
  File "C:\test\tts joiney\env\lib\site-packages\joinery\op.py", line 25, in process_to_file
    result = self.run_tts(chunk)
  File "C:\test\tts joiney\env\lib\site-packages\joinery\op.py", line 58, in run_tts
    return self.api.process_to_file(chunk)
  File "C:\test\tts joiney\env\lib\site-packages\joinery\api\openai.py", line 23, in process_to_file
    file_path = self.get_file_path(text)
  File "C:\test\tts joiney\env\lib\site-packages\joinery\api\base.py", line 24, in get_file_path
    f"{self.CACHE_DIR}/{self.service}_{self.model}_{self.voice}_{text_hash}.mp3"
AttributeError: 'OpenAIApi' object has no attribute 'CACHE_DIR'
```

Kindly Assign the issue to me.

**Comment by mayanklabs** [comment]
This PR addresses the concerns raised in this issue: #4 .

**Comment by drien** [comment]
Fixed in #4
